utilities/blob_db/blob_db_test.cc

```diff
@@ -25,7 +25,22 @@ class BlobDBTest : public testing::Test {
  public:
   const int kMaxBlobSize = 1 << 14;
 
-  BlobDBTest() : dbname_(test::TmpDir() + "/blob_db_test"), blob_db_(nullptr) {
+  class MockEnv : public EnvWrapper {
```

Comment:
Looking at the code, the way SpecialEnv mock NowMicros looks more useful for sleepping a period of time. Here I would like exact control of what NowMicros() return. Plus I don't need all other functionality of SpecialEnv. Shall we keep this one?